code/datums/dna/blocks/dna_features_block.dm

```diff
@@ -0,0 +1,167 @@
+/datum/dna_block/feature/mutant_color
+	block_length = DNA_BLOCK_SIZE_COLOR
+	feature_key = FEATURE_MUTANT_COLOR
+
+/datum/dna_block/feature/mutant_color/unique_block(mob/living/carbon/human/target)
+	. = ..()
```

Comment:
The parent calls on these `unique_block` procs are pretty un-ergonomic. 

From what I can tell all it's for is the stack trace - could that stack trace be moved to where the procs are invoked to make this easier to work with for future devs?